content/desktop/wsl/_index.md

```diff
@@ -51,6 +51,13 @@ Before you turn on the Docker Desktop WSL 2 feature, ensure you have:
 
 Now `docker` commands work from Windows using the new WSL 2 engine.
 
+>**Tip**
+>
+> By default, Docker Desktop stores the data for the WSL 2 engine at `C:\Users\[USERNAME]\AppData\Local\Docker\wsl`.
```

Comment:
Hard-coding C:\Users\ is a naïve and horrible idea when Windows already has an environment variable %USERPROFILE% which always works!

Reply:
The right env variable is %LOCALAPPDATA% and Path:
%LOCALAPPDATA%\Docker\wsl